src/chainrules.jl

```diff
@@ -0,0 +1,19 @@
+function rrule(::typeof(*), A::LinearMap, x::AbstractVecOrMat)
```

Comment:
It looks like the tests consider only the case where `x` is a vector.
Because `A * X` returns a new `LinearMap` here when `X` is a matrix, it is a bit unclear to me whether this will work, so I'd recommend either restricting it to vectors or explicitly testing the case where `x` is a matrix.